### DISCUSSION TITLE
多选控件与枚举类型，在表单控件中可以选择，在表格控件中不可选择

### DISCUSSION THREAD

**Discussion by yangf85** [dotnetcore/BootstrapBlazor#3192] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

表格编辑中
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/42733580/cd23dbbc-0f7f-4a04-ac85-5a7641314e28)
表单编辑
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/42733580/42458a3e-09db-48e6-a8e5-06a7ae239dfa)

### Steps To Reproduce

仓库地址
https://github.com/yangf85/BootstrapTest

**Comment by bb-auto** [comment]
@yangf85 Thank you for reporting. We will give triage later.

**Comment by ArgoZhang** [comment, marked as answer]
@yangf85 这个问题已经说过无数次了，参考文档链接如下：

https://www.blazor.zone/table/edit#EditTemplate

自定义如下代码如下：

```
@using static BootstrapTest.Server.Components.Pages.TableTest

<MultiSelect DisplayText="分类" @bind-Value=Categories />

@code {
    [Parameter]
    public List<SummaryCategory>? Categories { get; set; }

    [Parameter]
    public EventCallback<List<SummaryCategory>?> CategoriesChanged { get; set; }
}
```

原模板更改为 

```
<EditTemplate Context="item">
    <div class="col-12" >
        <Component @bind-Categories="context.Categories"></Component>
    </div>
</EditTemplate>
```

帮忙点一下下面的 `Mark as answer` 按钮

**Comment by yangf85** [comment]
哦。这个也要自定义模板啊，我以为只是级联的才需要

**Reply by ArgoZhang** [reply]
试一下，只要需要刷新的，都需要模板，不是多组件才需要模板，这个理解要到位
